gcsio/src/main/java/com/google/cloud/hadoop/gcsio/InvocationIdGrpcChannelInterceptor.java

```diff
@@ -0,0 +1,57 @@
+package com.google.cloud.hadoop.gcsio;
+
+import io.grpc.CallOptions;
+import io.grpc.Channel;
+import io.grpc.ClientCall;
+import io.grpc.ClientInterceptor;
+import io.grpc.ForwardingClientCall.SimpleForwardingClientCall;
+import io.grpc.Metadata;
+import io.grpc.MethodDescriptor;
+import java.util.UUID;
+
+public final class InvocationIdGrpcChannelInterceptor implements ClientInterceptor {
+
+  static final Metadata.Key<String> GOOG_API_CLIENT =
```

Comment:
add VisibleForTesting annotation?

Reply:
I am not sure if it is even required here. I am able to test the `private` functionality using the public functions.

Reply:
Not required since we made this package private